gatsby/src/components/BackToTop/BackToTop.module.scss

```diff
@@ -0,0 +1,15 @@
+@import '~@nice-digital/nds-core/scss/core';
+
+.backToTop {
+  background: red;
+  width: 100%;
+  margin-left: 0;
+  margin-bottom: -(rem($nds-spacing-large));
+}
+
+.fixed {
+  background: green;
+  position: fixed;
```

Comment:
Is there a way to use sticky like this: https://stackoverflow.com/a/54483332/486434 for browsers that support it?